src/dewey/config.clj

```diff
@@ -70,22 +70,22 @@
 (cc/defprop-optstr es-uri
   "The hostname for the Elasticsearch server"
   [props config-valid configs]
-  "dewey.es.uri" "http://elasticsearch:9200")
```

Comment:
As far as the defaults set by the service, we should probably retain this one's old value (which mimics having an `elasticsearch` service in kubernetes that's being connected to) and the `es-index` value below. Setting username & password to `admin` is probably fine and would make running this against a development OpenSearch easier, so those changes seem good to me.